src/frontend/js/product/ProductItem.spec.js

```diff
@@ -0,0 +1,43 @@
+import chai, { assert, expect } from 'chai';
+import { shallow } from 'enzyme';
+import chaiEnzyme from 'chai-enzyme';
+import React from 'react';
+
+import ProductItem from './ProductItem';
+
+
+describe('Component ProductItem', () => {
+    let props;
+
+    beforeEach(() => {
```

Comment:
Not needed, just initialize the props as a const at L10 and remove the `beforeEach` hook